service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceMethod.java

```diff
@@ -22,28 +23,47 @@
 {
     private IotHubConnectionString iotHubConnectionString = null;
     private Integer requestId = 0;
-    private static final int DEFAULT_RESPONSE_TIMEOUT = 30; // default response timeout is 30 seconds
-    private static final int DEFAULT_CONNECT_TIMEOUT = 0;
-    private static final int THOUSAND_MS = 1000;
+
+    private DeviceMethodClientOptions options;
+
     /**
      * Create a DeviceMethod instance from the information in the connection string.
      *
      * @param connectionString is the IoTHub connection string.
      * @return an instance of the DeviceMethod.
-     * @throws IOException This exception is thrown if the object creation failed
+     * @throws IOException This exception is thrown if the object creation failed.
      */
     public static DeviceMethod createFromConnectionString(String connectionString) throws IOException
+    {
+        return createFromConnectionString(connectionString, DeviceMethodClientOptions.builder()
```

Comment:
nit: can we have the tabbing differentiate between the builder args and createFromConnectionString args. Eg:
```java
public static DeviceMethod createFromConnectionString(String connectionString) throws IOException
{
    return createFromConnectionString(
            connectionString,
            DeviceMethodClientOptions.builder()
                    .httpConnectTimeout(DeviceMethodClientOptions.DEFAULT_HTTP_CONNECT_TIMEOUT_MS)
                    .httpReadTimeout(DeviceMethodClientOptions.DEFAULT_HTTP_READ_TIMEOUT_MS)
                    .build());
}
```

Reply:
I'll do it here and for the same block of code in DeviceTwin.java sure